### ISSUE TITLE
Utc class should be removed

### ISSUE THREAD

**Issue by Glusk2** [jcabi/jcabi-jdbc#78]
[Utc](https://github.com/jcabi/jcabi-jdbc/blob/9e72ffab8d25961e8d0012c45d5e12cefcae7f3d/src/main/java/com/jcabi/jdbc/Utc.java) should be removed from source.

According to this [SO answer](https://stackoverflow.com/a/49683937/6367213), SQL timestamps should be set and queried via the [Instant](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html) class:

> If you were using java.sql.Timestamp to exchange data with a database, no need for that class anymore. As of JDBC 4.2 and later, you can directly exchange java.time objects with your database.

``` java
myPreparedStatement.setObject( … , instant ) ;
```
...and…

``` java
Instant instant = myResultSet.getObject( … , Instant.class ) ;
```

It should be documented in the `README.md` that one should work with `Instant` objects whenever trying to set or query a `java.sql.Timestamp`.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@glusk2[/z](https://www.0crat.com/u/glusk2) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3RUBL5H9) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by andreoss** [comment]
@yegor256 What do you think? With better JSR310 support, `Utc` wouldn't be necessary .

**Comment by yegor256** [comment]
@Glusk @andreoss I agree. But we will have to release a new major version of jcab-jdbc, I believe.

**Comment by andreoss** [comment]
@Glusk 
After looking into the details it appears that
1) The current version of `pgjdbc` does not support `Instant` https://github.com/pgjdbc/pgjdbc/issues/1325
2) H2 does support Instant but only in the latest version 
3) MaridDB/MySQL Connector does not support `Instant`

**Comment by Glusk** [comment]
One could swap `pgjdbc` for: https://github.com/impossibl/pgjdbc-ng. It supports `Instant`s

Looking at [this page](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-versions.html), JDBC 4.2 spec seems to be supported by the most up to date Connector/J version.
